brainscore/benchmarks/zhu2019.py

```diff
@@ -0,0 +1,170 @@
+
+        self._metric = Accuracy()
+
+        super(_Zhu2019Accuracy, self).__init__(
+            identifier=f'yuille.Zhu2019_{dataset}-accuracy',
+            parent='yuille.Zhu2019',
+            ceiling_func=lambda: Score([1, np.nan], coords={'aggregation': ['center', 'error']}, dims=['aggregation']),
+            bibtex=BIBTEX, version=1)
+
+    def __call__(self, candidate: BrainModel):
+        categories = ["car", "aeroplane", "motorbike", "bicycle", "bus"]
+        candidate.start_task(BrainModel.Task.label, categories)
+        stimulus_set = place_on_screen(self._assembly.stimulus_set, target_visual_degrees=candidate.visual_degrees(),
+                                       source_visual_degrees=self._visual_degrees)
+        human_results = _human_assembly_categorical_distribution(self._assembly, collapse=False)
+        model_results = candidate.look_at(stimulus_set, number_of_trials=self._number_of_trials)
+
+        # compare model to human
+        raw_score = self._metric(model_results, human_results)
+
+        ceiling = calculate_ceiling(self._assembly, self._metric)
+        score = raw_score / ceiling
+        score.attrs['raw'] = raw_score
+        score.attrs['ceiling'] = ceiling
+        return score
+
+
+class _Zhu2019Accuracy_Engineering(BenchmarkBase):
+    # engineering benchmark: compares model to ground_truth
+    def __init__(self, dataset):
+        self._assembly = LazyLoad(lambda: load_assembly(dataset))
+        self._fitting_stimuli = brainscore.get_stimulus_set('yuille.Zhu2019_extreme_occlusion')
+        self._stimulus_set = LazyLoad(lambda: load_assembly(dataset).stimulus_set)
+        self._visual_degrees = 8
+        self._number_of_trials = 1
+
+        self._metric = Accuracy()
+
+        super(_Zhu2019Accuracy_Engineering, self).__init__(
+            identifier=f'yuille.Zhu2019_{dataset}-accuracy-engineering',
+            parent='yuille.Zhu2019',
+            ceiling_func=lambda: Score([1, np.nan], coords={'aggregation': ['center', 'error']}, dims=['aggregation']),
+            bibtex=BIBTEX, version=1)
+
+    def __call__(self, candidate: BrainModel):
+        categories = ["car", "aeroplane", "motorbike", "bicycle", "bus"]
+        candidate.start_task(BrainModel.Task.label, categories)
+        stimulus_set = place_on_screen(self._assembly.stimulus_set, target_visual_degrees=candidate.visual_degrees(),
+                                       source_visual_degrees=self._visual_degrees)
+        model_results = candidate.look_at(stimulus_set, number_of_trials=self._number_of_trials)
+        ground_truth = stimulus_set.sort_values("stimulus_id")["ground_truth"].values
+
+        # compare model with stimulus_set (ground_truth)
+        raw_score = self._metric(model_results, ground_truth)
+
+        ceiling = self.ceiling
+        score = raw_score / ceiling.sel(aggregation='center')
+        score.attrs['raw'] = raw_score
+        score.attrs['ceiling'] = ceiling
```

Comment:
ceiling is 1, so no need to ceil
```suggestion
        score = self._metric(model_results, ground_truth)
```